serverlessworkflow/modules/ROOT/pages/opentelemetry/otel-configuration.adoc

```diff
@@ -0,0 +1,403 @@
+= Configuring OpenTelemetry in {product_name}
+
+:compat-mode!:
+// Metadata:
+:description: Configuration guide for OpenTelemetry integration
+:keywords: kogito, workflow, serverless, opentelemetry, configuration, properties
+// Referenced documentation pages.
+:otel-core-concepts: xref:opentelemetry/otel-core-concepts.adoc
+
+This document describes how to configure OpenTelemetry for your {product_name} workflow applications.
+
+== Prerequisites
+
+Before configuring OpenTelemetry, ensure you have:
+
+* A {product_name} project set up
+* Understanding of {otel-core-concepts}[OpenTelemetry core concepts]
+* Access to an OpenTelemetry backend (Jaeger, Zipkin, OTLP endpoint, etc.)
+
+== Dependencies
+
+To enable OpenTelemetry in your {product_name} application, add the following dependency to your project:
+
+.Maven dependency
+[source,xml]
+----
+<dependency>
+    <groupId>org.apache.kie.sonataflow</groupId>
+    <artifactId>sonataflow-quarkus-otel</artifactId>
+</dependency>
+----
+
+.Gradle dependency
+[source,gradle]
+----
+implementation 'org.apache.kie.sonataflow:sonataflow-quarkus-otel'
+----
+
+[NOTE]
+====
+The OpenTelemetry extension is built on top of the Quarkus OpenTelemetry extension. Ensure your project includes the necessary Quarkus OpenTelemetry dependencies as well.
+====
+
+== Basic Configuration
+
+Configure OpenTelemetry by setting properties in your `src/main/resources/application.properties` file.
+
+=== Essential Configuration Properties
+
+.Basic OpenTelemetry configuration
+[source,properties]
+----
+# Enable OpenTelemetry
+quarkus.otel.enabled=true
+
+# Configure the service name
+quarkus.otel.service.name=my-workflow-service
+
+# Configure the OpenTelemetry exporter
+quarkus.otel.exporter.otlp.endpoint=http://localhost:4317
```

Comment:
this is confused, that's the default port for gRPC. 
As explained later, OTLP exporters use gRPC by default.
Even if the endpoint starts with http://, the protocol is still gRPC unless `quarkus.otel.exporter.otlp.protocol=http/protobuf `is set.

Reply:
This is the default value used in the quarkus doc: https://quarkus.io/guides/opentelemetry#quarkus-opentelemetry_quarkus-otel-exporter-otlp-endpoint that's why I used this value but I agree that having starts by grpc:// would make more sense as the default protocol is grpc but I value consistency with official doc more